js.go

```diff
@@ -1478,31 +1490,35 @@ func (p ReplayPolicy) MarshalJSON() ([]byte, error) {
 }
 
 var (
-	AckAck      = []byte("+ACK")
-	AckNak      = []byte("-NAK")
-	AckProgress = []byte("+WPI")
-	AckNext     = []byte("+NXT")
-	AckTerm     = []byte("+TERM")
+	ackAck      = []byte("+ACK")
+	ackNak      = []byte("-NAK")
+	ackProgress = []byte("+WPI")
+	ackNext     = []byte("+NXT")
+	ackTerm     = []byte("+TERM")
```

Comment:
I wasn't totally sure about this change. I didn't see a place that required the public user to pass any of these variables to us, so I decided to hide them instead.

Reply:
sounds good to me to keep them private.